backend/src/db/models/ClientCourses.ts

```diff
@@ -0,0 +1,19 @@
+import { Schema, model } from 'mongoose';
+
+import { IClientCourses } from 'interfaces/entities/IclientCourses';
+
+const clientCoursesSchema = new Schema<IClientCourses>({
+  _id: { type: Schema.Types.ObjectId },
+  title: { type: String, required: true },
+  description: { type: String },
+  technology: { type: String },
+  requiredSkills: [{ type: String }],
+  duration: { type: Number },
+  materials: [{ type: Schema.Types.ObjectId, ref: 'Materials' }],
+  testLink: { type: String },
+  status: { type: String },
+});
```

Comment:
client course no need to have title, description,technology and other field that comes from "Courses" collection as all you need it's to have a ref to "Course collection", in client course we should controll progress of taken courses  and other control over the course